src/main/java/io/outfoxx/swiftpoet/Util.kt

```diff
@@ -113,6 +113,8 @@ internal fun stringLiteralWithQuotes(
     return result.toString()
   } else {
     val result = StringBuilder(value.length + 32)
+    // using pre-formatted strings allows us to get away with not escaping symbols that would
+    // normally require escaping, e.g. "foo ${"bar"} baz"
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

```suggestion
    // Using pre-formatted strings allows us to get away with not escaping symbols that would
    // normally require escaping, e.g. "foo ${"bar"} baz".
```

Reply:
Was copy pasta https://github.com/square/kotlinpoet/blob/main/kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/Util.kt#L113